src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawable {
+    String getDrawing();

```

Comment:
This method should be void and print info about the figure directly
To 'draw' means to print out all information about a figure using System.out.println()